package.json

```diff
@@ -25,7 +25,7 @@
     "clean": "npx rimraf ./node_modules && cd source/native && rm -rf ./node_modules && cd .. && cd web && rm -rf ./node_modules",
     "clean:pods": "cd source/native/ios && rm -rf Pods",
     "install:pods": "cd source/native/ios && pod install",
-    "postinstall": "npx patch-package"
+    "postinstall": "npx patch-package && npx rimraf ./source/web/node_modules/react"
```

Comment:
What is rimraf?

Reply:
It's a library used to remove stuff. Here's the repo: https://github.com/isaacs/rimraf
We can replace this line with `rm -rf ./source/web/node_modules/react`